platform/fakes/fake_platform.go

```diff
@@ -197,8 +199,14 @@ func (p *FakePlatform) SetupSSH(publicKey, username string) error {
 	return p.SetupSSHErr
 }
 
-func (p *FakePlatform) SetUserPassword(user, encryptedPwd string) (err error) {
-	p.UserPasswords[user] = encryptedPwd
+func (p *FakePlatform) SetUserPassword(user string, encryptedPwd string) (err error) {
+	if user == boshsettings.VCAPUsername {
+		p.UserPassword = encryptedPwd
+	}
```

Comment:
@maximilien sure, in the status update call with Fabio last night, I mentioned you will stop by him to talk something about OS reload and others...:smile: ,  maybe plus different size issue for stemcell, they want to create an additional 100G stemcell, I am thinking if we could set stemcell size in deployment yaml like openstack....